configure.in

```diff
@@ -2254,6 +2254,11 @@ if test -z "$wx_cv_std_libfullpath"; then
     wx_cv_std_libfullpath="/usr/$wx_cv_std_libpath"
 fi
 
+dnl This is Solaris, and needs its pkg-config path adjusted for 64-bit.
```

Comment:
Should we perhaps do this in the `case` above? If we really want to do it only for Solaris and not for any system using `/usr/lib/64` (I realize that currently it's equivalent), it would seem more logical.

Reply:
`$wx_cv_std_libfullpath` does not yet have a value in the `case` above, which is why I placed this here. But on the other hand, the libpath's prefix in the case of Solaris can only by `/usr` as far as I can see, so one might as well just use the absolute `/usr/$wx_cv_std_libpath` within the `case`.